axeDevToolsXCUI.xcframework/ios-arm64_x86_64-simulator/axeDevToolsXCUI.framework/_CodeSignature/CodeResources

```diff
@@ -47,7 +47,7 @@
 		<dict>
 			<key>hash2</key>
 			<data>
-			f1x5dmIW1l1DaSk5jT+6XQC3U8UHlNF9yZsJPz0xUNY=
+			ikAvuWZRo7HDSxPdVdlB0iPOCr6pR2hdOfd83ZX8IPo=
```

Comment:
Just curious since I'm not used to seeing our stuff in this format - how does this <data> item get generated? Again, just completely out of curiosity.